### DISCUSSION TITLE
AudioGridder plugin crashes upon initialization

### DISCUSSION THREAD

**Discussion by qwag** [apohl79/audiogridder#1252] [Help]
I was using a plugin within AudioGridder and noticed some bad performance, so I changed my sample rate from 48000 to 44100. AudioGridder now crashes anytime I load it in a sample rate other than 48000.
There is a similar discussion post here, but there wasn't a real solution. Surely there's a way to use this plugin at a sample rate of 44100.

![CleanShot 2023-07-24 at 01 17 19@2x](https://github.com/apohl79/audiogridder/assets/82276858/1441c896-87e6-46da-b1ab-9e9b7fd085cd)

**Comment by ktamas77** [comment]
Confirm the AuV2 plugin is also crashing, unless you set the sample rate to 48khz. My system is currently running on 44.1khz, is there a chance that AudioGridder will run on 44.1khz as well?

**Comment by qwag** [comment]
I tried downgrading to the v1.1.1 plugin, and that works on 44100. 

It's still not a solution, but a temporary fix while the stability of 1.2.0 is being worked on, I suppose.
